artifactory/commands/transferfiles/transfer.go

```diff
@@ -750,3 +750,10 @@ func parseErrorsFromLogFiles(logPaths []string) (allErrors FilesErrors, err erro
 	}
 	return
 }
+
+func assertSupportedTransferDirStructure() error {
+	if err := state.VerifyTransferRunStatusVersion(); err != nil {
+		return err
+	}
+	return assertRepositorySpecificStructure()
```

Comment:
What is the reason we're running the `assertRepositorySpecificStructure ` check? Isn't ti enough running the `VerifyTransferRunStatusVersion`?